openai_agents/workflows/tools_workflow.py

```diff
@@ -3,13 +3,10 @@
 from datetime import timedelta
 
 from temporalio import workflow
-from temporalio.contrib.openai_agents.temporal_tools import activity_as_tool
+from temporalio.contrib import openai_agents as temporal_agents
+from agents import Agent, Runner
 
-# Import our activity, passing it through the sandbox
-with workflow.unsafe.imports_passed_through():
```

Comment:
I think all the files in this dir can remove this `with workflow.unsafe.imports_passed_through():` part now that we implicitly pass through `agents` module